modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -194,6 +194,9 @@ https://anypoint.mulesoft.com/accounts/api/profile
 </settings>
 ----
 
+[IMPORTANT]
+`<username>` needs to be kept as in the example given that is used by the platform to identify the token will be used. You need to set your token in `<password>ACCESS_TOKEN</password>` tag.
```

Comment:
Do not change `<username>~~~TOKEN</username>`, because that value tells the platform that you're using a token. Set your token in `<password>ACCESS_TOKEN</password>`.

Reply:
https://www.mulesoft.org/jira/browse/DOCS-3961